prestoadmin/util/validators.py

```diff
@@ -31,18 +31,18 @@ def validate_username(username):
 
 
 def validate_port(port):
+    if not isinstance(port, basestring):
```

Comment:
I think we should accept either a string or an integer, we want to avoid times when the user is unexpectedly told things don't work